import-api/mastr_storage.json

```diff
@@ -0,0 +1,162 @@
+    "grantNo": "unknown"},
+"spatial":
+    {"location": "none",
+    "extent": "Germany",
+    "resolution": "vector"},
+"temporal":
+    {"referenceDate": "2019-01-31",
+    "start": "none",
+    "end": "none",
+    "resolution": "none"},
+"sources": [
+    {"title": "Bundesnetzagentur - Marktstammdatenregister", "description": "Das Marktstammdatenregister ist das Register für den deutschen Strom- und Gasmarkt. Es wird MaStR abgekürzt. Im MaStR sind vor allem die Stammdaten zu Strom- und Gaserzeugungsanlagen zu registrieren. Außerdem sind die Stammdaten von Marktakteuren wie Anlagenbetreibern, Netzbetreibern und Energielieferanten zu registrieren. Das MaStR wird von der Bundesnetzagentur geführt.", "path": "https://www.marktstammdatenregister.de/MaStR/", "license": "Datenlizenz Deutschland – Namensnennung – Version 2.0", "copyright": "© Marktstammdatenregister 2019 | dl-de/by-2-0"},
+    {"title": "RLI - open_MaStR", "description": "Scripts to download, process and publish the MaStR data set.", "path": "https://github.com/OpenEnergyPlatform/data-preprocessing/", "license": "GNU Affero General Public License Version 3 (AGPL-3.0)", "copyright": "© Reiner Lemoine Institut"},
+    {"title": "Logfile", "description": "Logging", "path": "mastr.log", "license": "CC0-1.0", "copyright": "none"}],
+"licenses": [
+    {"name": "dl-de/by-2-0",
+    "title": "Datenlizenz Deutschland – Namensnennung – Version 2.0",
+    "path": "https://www.govdata.de/dl-de/by-2-0",
+    "instruction": "Die bereitgestellten Daten und Metadaten dürfen für die kommerzielle und nicht kommerzielle Nutzung verwendet werden. Bei der Nutzung ist sicherzustellen, dass Angaben als Quellenvermerk enthalten sind.",
+    "attribution": "© Marktstammdatenregister 2019 | dl-de/by-2-0"}],
+"contributors": [
+    {"title": "Izzet", "email": "none", "date": "2019-10-02", "object": "metadata", "comment": "Create metadata"}],
+"resources": [
+    {"profile": "tabular-data-resource",
+    "name": "supply.bnetza_mastr_storage",
+    "path": "bnetza_mastr_storage.csv",
+    "format": "csv",
+    "encoding" : "UTF-8",
+    "schema": {
+        "fields": [
+            {"name": "EinheitMastrNummer", "description": "MaStR-Nummer der Einheit", "type": "string", "unit": "none"},
+            {"name": "id", "description": "ID der Einheit", "type": "int", "unit": "none"},
+            {"name": "lid", "description": "LID der Einheit", "type": "int", "unit": "none"},
+            {"name": "Name", "description": "Name der Einheit", "type": "string", "unit": "none"},
+            {"name": "Einheitart", "description": "Art der Einheit", "type": "string", "unit": "none"},
+            {"name": "Einheittyp", "description": "Typ der Einheit", "type": "string", "unit": "none"},
+            {"name": "Standort", "description": "Standort der Einheit", "type": "string", "unit": "none"},
+            {"name": "Bruttoleistung", "description": "Bruttoleistung der Einheiten", "type": "float", "unit": "none"},
```

Comment:
unit is not none. It's "kW".

Reply:
@Izzet91 did it in 8fd50a32a5e540c143e9de5258dd15051279de31